scio-bigquery/src/test/scala/com/spotify/scio/bigquery/types/TypeProviderTest.scala

```diff
@@ -28,12 +29,6 @@ import scala.reflect.runtime.universe._
 object TypeProviderTest {
   @BigQueryType.toTable
   case class RefinedClass(a1: Int)
-}
-
-// TODO: mock BigQueryClient for fromTable and fromQuery
-class TypeProviderTest extends FlatSpec with Matchers {
-
-  val NOW = Instant.now()
 
   @BigQueryType.fromSchema(
```

Comment:
I moved these classes to the companion object because these annotated classes  should be in a companion object (?)